From c77ffa275c776758dd125e32d03ff8c7f434e788 Mon Sep 17 00:00:00 2001
From: Edmunt Pienkowsky <roed@onet.eu>
Date: Fri, 11 Dec 2020 10:50:36 +0100
Subject: [PATCH] Increase response timeouts


diff --git a/temper.py b/temper.py
index 38f5376..8757731 100755
--- a/temper.py
+++ b/temper.py
@@ -146,7 +146,7 @@ class USBRead(object):
 
       firmware = b''
       while True:
-        r, _, _ = select.select([fd], [], [], 0.2)
+        r, _, _ = select.select([fd], [], [], 2)
         if fd not in r:
           break
         data = os.read(fd, 8)
@@ -180,7 +180,7 @@ class USBRead(object):
     os.write(fd, struct.pack('8B', 0x01, 0x80, 0x33, 0x01, 0, 0, 0, 0))
     bytes = b''
     while True:
-      r, _, _ = select.select([fd], [], [], 0.1)
+      r, _, _ = select.select([fd], [], [], 2)
       if fd not in r:
         break
       data = os.read(fd, 8)
